lib/whenever/cron.rb

```diff
@@ -154,6 +150,17 @@ def parse_as_string
         raise ArgumentError, "Couldn't parse: #{@time}"
       end
 
+      def range_or_integer(at, valid_range, name)
+        must_be_between = "#{name} must be between #{valid_range.min}-#{valid_range.max}"
+        unless at.is_a?(Range)
+          raise ArgumentError, "#{must_be_between}, #{at} given" unless valid_range.include?(at)
```

Comment:
I’d prefer to invert this conditional.

Reply:
I've changed it to `if !at.is_a?(Range)`, assuming that's what you meant.